### PR TITLE
Leaderboard project:Project Setup

### PR DESCRIPTION
**Pull request by Tumworobere**
In this project, I achieved the following

- Used ES6 modules, with import and export
- Created an npm project with webpack
- Wrote plain HTML markup with minumum styling
- Used Gitflow in this project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "babel-jest": "^27.3.1",
+    "jest": "^27.3.1",
+    "ts-jest": "^27.0.7"
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.25.3",
+    "hint": "^6.1.9",
```
Comment (SajjadAhmad14) [inline package.json:19]: Please add `webpack, webpack-dev-server` to dev dependencies.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
```
Comment (SajjadAhmad14) [inline README.md:2]: Please add `npm install` and `npm start` commands in README as the project can not be set up locally without these commands.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^4.9.1"
+  },
+  "scripts": {
+    "test": "jest",
+    "start": "node server.js",
+    "build": "webpack"
+  },
```
Comment (SajjadAhmad14) [inline package.json]: I am unable to use the `npm start` command as it gives an error. Please update `scripts` as:
`
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
`.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "server.js",
+  "dependencies": {
```
Comment (SajjadAhmad14) [inline package.json:6]: Please use webpack to bundle JavaScript.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Tumworobere, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git Flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: package.json (hunk 3 above)
Comment (Tumworobere) [reply package.json]: Thank you @SajjadAhmad14 I have worked on all the request changes!

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @Tumworobere  great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**